### PR TITLE
docs(maps): jupyter notebook now auto-updates docs site

### PR DESCRIPTION
**Pull request by rusackas**
### SUMMARY

So we have this Jupyter notebook to generate maps, and typescript for the control panel.

Now, it also outputs JSON to the docs site, and the docs site in turn renders the JSON with React (in three columns!) to keep an ever-current list of countries available on the docs.

### BEFORE/AFTER SCREENSHOTS OR ANIMATED GIF

Before:
image

After:
image

### ADDITIONAL INFORMATION

- [ ] Has associated issue:
- [ ] Required feature flags:
- [ ] Changes UI
- [ ] Includes DB Migration (follow approval process in [SIP-59](https://github.com/apache/superset/issues/13351))
  - [ ] Migration is atomic, supports rollback & is backwards-compatible
  - [ ] Confirm DB migration upgrade and downgrade tested
  - [ ] Runtime estimates and downtime expectations provided
- [ ] Introduces new feature or API
- [ ] Removes existing feature or API

### CODE REVIEW

**Review by mistercrunch — APPROVED** [review]

**Review by mistercrunch — APPROVED** [review]

**Comment by rusackas** [comment]
Thanks for the review... just need a maps codeowner stamp from @villebro

**Review by mistercrunch — APPROVED** [review]

**Comment by rusackas** [comment]
@villebro I just need a notebook codeowner stamp, and we'll be all good :)

**Comment by rusackas** [comment]
@mistercrunch we can make you a codeowner too for the notebook, if you promise to guard against map PRs being made without notebook updates

**Review by villebro — APPROVED** [review]
Nice!

**Comment by rusackas** [comment]
I keep getting this error, so I'm setting FetchDepth = 0 here. Hope that doesn't cause issues (i.e. perf problems) elsewhere.

File: .github/workflows/superset-frontend.yml [hunk 1]
```diff
@@ -20,6 +20,7 @@ jobs:
       - name: "Checkout ${{ github.ref }} ( ${{ github.sha }} )"
         uses: actions/checkout@v4
         with:
+          fetch-depth: 0 # Fetch all history
```
Comment (mistercrunch) [inline .github/workflows/superset-frontend.yml:23]: hey, curious why this was/is neeeded?

Comment (rusackas) [reply .github/workflows/superset-frontend.yml:23]: It was indeed to get around this:

image

**Pull request merged**